sdk/policyinsights/Microsoft.Azure.Management.PolicyInsights/src/Generated/IPolicyStatesOperations.cs

```diff
@@ -53,7 +53,7 @@ public partial interface IPolicyStatesOperations
         /// <exception cref="Microsoft.Rest.ValidationException">
         /// Thrown when a required parameter is null
         /// </exception>
-        Task<AzureOperationResponse<PolicyStatesQueryResults>> ListQueryResultsForManagementGroupWithHttpMessagesAsync(string policyStatesResource, string managementGroupName, QueryOptions queryOptions = default(QueryOptions), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
+        Task<AzureOperationResponse<IPage<PolicyState>>> ListQueryResultsForManagementGroupWithHttpMessagesAsync(string policyStatesResource, string managementGroupName, QueryOptions queryOptions = default(QueryOptions), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
This is a breaking change, correct?  This is good, but we need to bump the major version

Reply:
ok, i will change the major version too.

Reply:
No, it is not a breaking change from API point of view.

Reply:
It's a breaking change from the api point of view becauser the returned type has changed, and does not have the same properties and methods as the original type (IP{age<T> versus PolicyStatesQueryResults